### ISSUE TITLE
Custom Argument Resolving gets interrupted by Doctrine

### ISSUE THREAD

**Issue by Sengorius** [symfony/symfony-docs#8589]
I just create my own argument resolver for the user entity that is used in one of our projects and did this like explained [in the official documentation](https://symfony.com/doc/current/controller/argument_value_resolver.html).

The class seem to work well, but Doctrine was faster to put one of his resolvers in between, so I only got an exception: `Unable to guess how to get a Doctrine instance from the request information for parameter "user".`

I found out that some configuration of the FrameworkExtraBundle was needed:
```yaml
# app/config/config.yml
sensio_framework_extra:
    request:
        converters: true
        auto_convert: false
```

This can be found in the [ParamConverter documentation](https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/converters.html), but a hint in the upper docs would be nice for upcoming developers. :wink:

**Comment by jtreminio** [comment]
Thank you @Sengorius, this is indeed missing from the docs and the example does not work without this config block.

**Comment by kshishkin** [comment]
Custom Argument Resolver documentation is another good place this information would be valuable:
https://symfony.com/doc/current/controller/argument_value_resolver.html
I think it could be explicitly stated as a conflict between argument resolver & param converter.

**Comment by javiereguiluz** [comment]
Fixed by #10099.
